x-pack/plugins/security_solution/public/users/pages/details/index.tsx

```diff
@@ -64,11 +67,11 @@ const UebaDetailsComponent: React.FC<UebaDetailsProps> = ({ detailName, uebaDeta
   const { globalFullScreen } = useGlobalFullScreen();
```

Comment:
I don't know when `useGlobalFullScreen` returns true. I tried making the table fullScreen and opening the timeline, and the value didn't change in the store. Do you know why it is for?

Reply:
it's used by analyzer to enter fullscreen, look for usages of the other part of this hook, setGlobalFullScreen. Alot of the pages in the app make use of this `<Display show={!globalFullScreen} />` component to hide things when the app is in fullscreen, which is not necessary anymore and just making the browser do unnecessary work. I was planning on removing uses of it as part of some upcoming tech debt work